.github/docker/ubuntu-20.04.Dockerfile

```diff
@@ -23,13 +23,13 @@ ARG BASE_DEPS="\
 	git"
 
 # UMF's dependencies
-ARG UMF_DEPS="\
-	libhwloc-dev \
-	libtbb-dev"
+# libhwloc-dev is required
```

Comment:
this comment should be a little more verbose - saying that it's installed here via script, as the hwloc version is too old on this OS

and please move `HWLOC_DEPS` and `# Copy hwloc` here, above.